CMSgov/mint-app
# Easi 2788/audit deleted actor implementation

# EASI-2788

## Changes and Description

- All delete statements are wrapped in transactions
   - first part of the transaction sets the current user scope



## How to test this change
1. Create objects and delete them. 
2. Look at the audit logs, verify that you see an audit entry that records the correct user who deleted the entry.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
